### PR TITLE
Create ui-c8y-1020-28-8-no-possibility-to-delete-devices-in-child-devices-grid.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [DM-3951](https://cumulocity.atlassian.net/browse/DM-3951)
Original PR: [7193](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7193)

### CODE REVIEW

File: content/change-logs/device-management/ui-c8y-1020-28-8-no-possibility-to-delete-devices-in-child-devices-grid.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: No possibility to delete devices in Child devices grid
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1020-28-8-no-possibility-to-delete-devices-in-child-devices-grid.md]: The Claude AI suggested to use the following title:
```suggestion
title: Enable deletion of child devices in the device management
```

File: content/change-logs/device-management/ui-c8y-1020-28-8-no-possibility-to-delete-devices-in-child-devices-grid.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3951
+version: 1020.28.8
+---
+No possibility to delete devices in Child devices grid
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1020-28-8-no-possibility-to-delete-devices-in-child-devices-grid.md]: The Claude AI suggested to use the following body:
```suggestion
In the device management, it was previously not possible to delete child devices from the grid view, requiring users to navigate to the device details page for deletion. With this change, child devices can now be directly deleted from the grid view in the same way as regular devices. This improvement simplifies the device management process and saves time for users with large device hierarchies.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/device-management/ui-c8y-1020-28-8-no-possibility-to-delete-devices-in-child-devices-grid.md (hunk 1 above)
Comment (bozhidardedov) [inline content/change-logs/device-management/ui-c8y-1020-28-8-no-possibility-to-delete-devices-in-child-devices-grid.md]:
```suggestion
title: Enable deletion of child devices in Device Management
```

File: content/change-logs/device-management/ui-c8y-1020-28-8-no-possibility-to-delete-devices-in-child-devices-grid.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
```
Comment (bozhidardedov) [inline content/change-logs/device-management/ui-c8y-1020-28-8-no-possibility-to-delete-devices-in-child-devices-grid.md]:
```suggestion
In Device Management, the action to delete child devices from the "Child devices" tab was removed, requiring users to navigate to the device details page for deletion. With this change, child devices can now be directly deleted from the "Child devices" grid view.
```

**Review by aaronraab — APPROVED** [review]

File: content/change-logs/device-management/ui-c8y-1020-28-8-no-possibility-to-delete-devices-in-child-devices-grid.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3951
+version: 1020.28.8
+---
+In Device Management, the action to delete child devices from the "Child devices" tab was removed, requiring users to navigate to the device details page for deletion. With this change, child devices can now be directly deleted from the "Child devices" grid view.
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1020-28-8-no-possibility-to-delete-devices-in-child-devices-grid.md]:
```suggestion
In the Device Management application, child devices could not be deleted from the **Child devices** tab. Instead, users had to to navigate to the device details for deletion. With this change, child devices can now be directly deleted from the **Child devices** tab.
```

File: content/change-logs/device-management/ui-c8y-1020-28-8-no-possibility-to-delete-devices-in-child-devices-grid.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Enable deletion of child devices in Device Management
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1020-28-8-no-possibility-to-delete-devices-in-child-devices-grid.md]:
```suggestion
title: Enabled deletion of child devices from the Child devices tab
```

**Review by BeateRixen — APPROVED** [review]

**Review by dafkk — APPROVED** [review]

**Pull request merged**